azerothcore/azerothcore-wotlk
# fix(ICC/Deathbringer Saurfang) NPC Muradin Dialogues

## Changes Proposed:

This PR proposes changes to:

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/8921
- Closes https://github.com/azerothcore/azerothcore-wotlk/pull/13506

## Tests Performed:

This PR has been:

![WoWScrnShot_052024_041835](https://github.com/azerothcore/azerothcore-wotlk/assets/2810187/2f8f42f6-6e75-4c7d-bd1b-e8a392a726f7)

![WoWScrnShot_052024_041928](https://github.com/azerothcore/azerothcore-wotlk/assets/2810187/4ce532e5-6212-417c-8e92-6e2324ae9021)

## How to Test the Changes:


1. `.tele IcecrownCitadel`
2. Eliminate the first 3 bosses, and then start the fourth fight. Without the pull request, the script is not updated, and the dialogs are displayed incorrectly. I don't know if there is another way to do it, but I looked for thousands of ways, and the only one that occurred to me, as a final point, to fix it, until a better way appears, was this.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
